src/components/DashboardPanel/DashboardPanel.js

```diff
@@ -12,8 +13,8 @@ export default class DashboardPanel extends Component {
     super(props);
     this.state = {
       dataModel: [],
-      startDate: '2018-03-19',
-      endDate: '2018-03-24',
+      startDate: moment(this.props.startDate).format('YYYY-MM-DD'),
```

Comment:
Abstract the date format so that we are not using hard coded strings in multiple places